charts/nx-cloud/templates/_helpers.tpl

```diff
@@ -182,7 +182,7 @@ volumes:
   - emptyDir: {}
     name: cacerts
   - configMap:
-      name: {{ $selfSigned }}
+      name: {{ default $selfSigned .Values.preBuiltJavaCertStoreConfigMap }}
```

Comment:
to make it backwards compatible, we are just adding the *new* option in here, and we are using that only if they defined it

to be clear, they'll still use `$selfSigned`, but if the cert imports are slow for Java, they can also define this new option in addition to the old one
we'll always need `$selfSigned` (even if they defined `preBuiltJavaCertStoreConfigMap`) because the frontend pod expect a list of certs to load in (and it's fast)